src/imp/cgroup.c

```diff
@@ -168,5 +169,41 @@ struct cgroup_info *cgroup_info_create (void)
     return cgroup;
 }
 
+int cgroup_kill (struct cgroup_info *cgroup, int sig)
+{
+    int count = 0;
+    int rc = 0;
+    int saved_errno = 0;
+    char path [PATH_MAX+15]; /* cgroup->path + "/cgroup.procs" */
```

Comment:
Did you mean `PATH_MAX+1`?

Reply:
hm. cgroup->path is a buffer of size `PATH_MAX`. We're going to write 14 more characters so we need at least that much space or the compiler complains. Happy to do this another way.

Reply:
Oh right, I hate that.  Oh well.